tests/test_paper.py

```diff
@@ -0,0 +1,164 @@
+import trio
+import pytest
+import tractor
+import math
+import os
+from shutil import rmtree
```

Comment:
as [mentioned above](https://github.com/pikers/piker/pull/462/files#r1109032099) we've been trying to keep the import grouping consistent 😉